spec/atm_spec.rb

```diff
@@ -66,6 +61,15 @@
       subject.do_transaction(5, account)
       expect(subject.funds).to eq 995
     end
+
+    it 'does not allow for withdraw of 105$ from ATM if money not avilable in person account' do
```

Comment:
Metrics/LineLength: Line is too long. [96/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)